docker-build-package

```diff
@@ -16,7 +16,7 @@
 
 set -e
 
-IMAGE_NAME_PREFIX=mendersoftware/mender-dist-packages:builder
+IMAGE_NAME_PREFIX=mendersoftware/mender-test-containers:mender-dist-packages-builder
```

Comment:
I understand that there is a reason that we are using mender-test-containers with a tag, and not a dedicated image?

Reply:
The reason is to be consistent with every other image built in `mender-test-containers` repository, as listed [here](https://github.com/mendersoftware/mender-test-containers/#mender-test-containers).